src/graphql/error-map.ts

```diff
@@ -549,6 +563,8 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
     case "KratosError":
     case "AuthenticationKratosError":
     case "ExtendSessionKratosError":
+    case "PhoneAccountAlreadyExists":
+    case "PhoneAccountAlreadyExistsNeedToSweepFunds":
```

Comment:
Can you map these to a differentiated error, so that the frontend is capable of parsing it? `PhoneAccountAlreadyExistsError` could be the name.

Reply:
https://github.com/GaloyMoney/galoy/pull/2071/commits/93a078bc5207eb4432150d47f80bcd4657c7dd22